src/OneLogger.php

```diff
@@ -42,7 +42,7 @@ class OneLogger extends Plugin
   /**
    * @var string
    */
-  public $schemaVersion = '2.0.0';
+  public $schemaVersion = '2.0.1';
```

Comment:
We don't need to increment this one because the DB schema didn't change.

Reply:
Done! Thanks for considering!